checkout/src/test/java/com/checkout/base/model/CountryTest.kt

```diff
@@ -42,29 +40,20 @@ internal class CountryTest {
     }
 
     @Test
-    fun `given country details should returns invalid country`() {
-        // Given
-        val expectedCountry = Country.INVALID_COUNTRY
-
+    fun `given country details should returns null`() {
```

Comment:
Shall we state that `Iso3166Alpha2` is `null` in the title?

Reply:
Sure